projects/transactions-journey/src/lib/communication/index.ts

```diff
@@ -0,0 +1,11 @@
+import { InjectionToken } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Transaction } from '../model/transaction';
+
+export interface TransactionsCommunicationService {
+  latestTransaction$: Observable<Transaction | undefined>;
```

Comment:
I prefer to have a default value for such things. Do you think it makes something inconvinient?

Reply:
from where I see, if I call this service is because I will have a transaction anyway, right? if so, I can just not expect undefined. Also it can reduce a lot of validations with that assumption further

Reply:
Not really, I expect a long-living stream, that emits new value for every new transaction, but at the very beginning, there is no newest transaction, this is why the value is undefined. This is how i read the sense of it.